src/go/rpk/pkg/config/config.go

```diff
@@ -28,7 +28,7 @@ const (
 
 	DefaultKafkaPort     = 9092
 	DefaultSchemaRegPort = 8081
-	DefaultProxyPort     = 8082
+	DefaultProxyPort     = 7071
```

Comment:
I think we shouldn't change this here, instead, change it in the `container start` command only.